classes/Infrastructure/Auth/CsrfCheck.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace OpenCFP\Infrastructure\Auth;
+
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Security\Csrf\CsrfToken;
+use Symfony\Component\Security\Csrf\CsrfTokenManager;
+
+class CsrfCheck
+{
+    /**
+     * @var CsrfTokenManager
+     */
+    private $manager;
+
+    public function __construct(CsrfTokenManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    public function checkCsrf($tokenId, $token)
```

Comment:
Can we add type declarations here, and if these are not sufficient, add a docblock as well?